### PR TITLE
Admins can now change users roles.

### LINKED ISSUE #87: Add way to make person admin/mentor

### PR DESCRIPTION
**Pull request by agundy**
Fixes  #87.

### CODE REVIEW

File: client/app/user/profile/profile.controller.js [hunk 1]
```diff
@@ -8,10 +8,13 @@ angular.module('observatory3App')
       Auth.getCurrentUser(function(user){
         loggedInUser = user;
         $scope.isuser = loggedInUser._id === $stateParams.id;
+        $scope.loggedInUserRole = loggedInUser.role;
       });
 
       $http.get('/api/users/' + $stateParams.id).success(function(user){
           $scope.user = user;
+          $scope.originalRole = user.role;
+          console.log($scope.originalRole);
```
Comment (seveibar) [inline client/app/user/profile/profile.controller.js]: log line

Comment (agundy) [inline client/app/user/profile/profile.controller.js]: good catch

**Comment by seveibar** [comment]
Can users be admins and mentors? I see a possible problem arising...

**Comment by seveibar** [comment]
I think it's fine as long as we only have the 3 roles "user","mentor" and "admin", but if we have more we're going to have to do this in a more "tag-like" fashion

**Comment by agundy** [comment]
I think we can assume if someone is an admin they are also some form of mentor

**Comment by seveibar** [comment]
right, but we may want to introduce a new role e.g. faculty

**Comment by seveibar** [comment]
like i said, it's fine for now, but possibly an issue later

**Comment by agundy** [comment]
Agreed. But i just built this off our existing work. I don't think a refactor of role is a concern right now.

**Comment by seveibar** [comment]
after the console.log change :shipit:

**Comment by seveibar** [comment]
I agree

**Pull request merged**